ports/javascript/mpconfigport.h

```diff
@@ -210,3 +210,9 @@ typedef long mp_off_t;
 
 #define MICROPY_PORT_ROOT_POINTERS \
     const char *readline_hist[8];
+
+#ifdef FROZEN_MPY_DIR
```

Comment:
This part isn't needed.  Please see how other ports (eg stm32, esp8266).  It can all be done in the Makefile.